### PR TITLE
Add bin/start_tileserver.sh

### PR DESCRIPTION
**Pull request by kaidamasaki**

### CODE REVIEW

File: bin/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+cd "$BASEDIR"
+JARFILE="$(ls -rt target/scala-*/tileserver-assembly-*.jar 2>/dev/null | tail -n 1)"
+SRC_PATHS=($(find .  -maxdepth 2 -name 'src' -o -name '*.sbt' -o -name '*.scala'))
+if [ -z "$JARFILE" ] || find "${SRC_PATHS[@]}" -newer "$JARFILE" | egrep -q -v '(/target/)|(/bin/)'; then
+    if [ "$1" == '--fast' ]; then
+        echo 'Assembly is out of date.  Fast start is enabled so skipping rebuild anyway...' >&2
+    else
+        nice -n 19 sbt assembly >&2
+        JARFILE="$(ls -rt target/scala-*/tileserver-assembly-*.jar 2>/dev/null | tail -n 1)"
+        touch "$JARFILE"
```
Comment (NathalieKaligirwa) [inline bin/build.sh]: Running `touch "$JARFILE"` from the terminal fails but it works fine from the script.

Comment (kaidamasaki) [inline bin/build.sh]: ...I'm confused by this comment.  Did you set `$JARFILE` from the correct directory?

Comment (NathalieKaligirwa) [inline bin/build.sh]: I did. I am probably doing something wrong. But the script itself works fine, it was an observation :)

**Comment by NathalieKaligirwa** [comment]
lgtm 🚢 it!

File: bin/build.sh (hunk 1 above)
Comment (kaidamasaki) [inline bin/build.sh]: I can maybe help you figure out what you did wrong in the terminal if you'd like, but it doesn't seem urgent.

**Pull request merged**